packages/dds/tree/src/test/shared-tree/fuzz/fuzzEditGenerators.ts

```diff
@@ -460,6 +520,32 @@ export const makeTransactionEditGenerator = (
 	};
 };
 
+export const makeSchemaEdit = (
+	opWeights: Partial<EditGeneratorOpWeights>,
+): Generator<FuzzSchemaChange, FuzzTestState> => {
+	const passedOpWeights = {
+		...defaultEditGeneratorOpWeights,
+		...opWeights,
+	};
+	const makeSchemaOp = (state: FuzzTestState) => {
+		return { type: "schema", contents: { type: state.random.uuid4() } };
+	};
+	const schemaType = createWeightedGenerator<FuzzSchemaOp, FuzzTestState>([
+		[makeSchemaOp, passedOpWeights.schema],
+	]);
```

Comment:
this generator doesn't do anything. You can just inline the `contents` field you generate above.

Reply:
In-lined contents field and removed unnecessary generator